### PR TITLE
refactor : Remove unwanted area specific `editorconfig`.

### PR DESCRIPTION
**Pull request by ShreyasJejurkar**
1. Removed `.editorconfig` for Http area and pulled rule CA1309 to top level editorconfig and fixed the violations. 

WIP : checking other area `.editorconfig` files

### CODE REVIEW

**Comment by wtgodbe** [comment]
Thanks, this looks like a good change! @adityamandaleeka do you know of any reason we'd need to keep the area-specific `.editorconfig` in HTTP? Looks like it hasn't been touched since Kevin added it 3 years ago.

@ShreyasJejurkar are you still iterating on this PR, or would you like me to review it now?

**Comment by ShreyasJejurkar** [comment]
Hi @wtgodbe, am checking other areas editorconfig as well, I guess we can hold this off as of now.

Having office work, so not getting time to look it. But don't worry, I will get this completed.

File: src/SignalR/clients/csharp/.editorconfig [hunk 1]
```diff
@@ -1,4 +0,0 @@
-[*.{cs,vb}]
-
-# CA2007: Consider calling ConfigureAwait on the awaited task
-dotnet_diagnostic.CA2007.severity = warning
```
Comment (BrennanConroy) [inline src/SignalR/clients/csharp/.editorconfig:4]: ?

**Comment by mitchdenny** [comment]
This is an old PR with merge conflicts ... should it be closed?

**Comment by captainsafia** [comment]
Putting the needs author input label on this to get insights from @ShreyasJejurkar.

**Pull request closed without merging**